.test-infra/jenkins/job_beam_PreCommit_Java_MavenInstall.groovy

```diff
@@ -44,7 +44,7 @@ mavenJob('beam_PreCommit_Java_MavenInstall') {
   goals('''\
     --batch-mode \
     --errors \
-    --activate-profiles release,jenkins-precommit,direct-runner,dataflow-runner,spark-runner,flink-runner,apex-runner \
+    --activate-profiles release,direct-runner,dataflow-runner,spark-runner,flink-runner,apex-runner \
```

Comment:
These profiles are only for the examples to run against different runners. I think there should be either none or just `direct-runner`.